src/domain/authentication/refreshAuth.ts

```diff
@@ -0,0 +1,10 @@
+import authService from './authService';
+import isValidToken from './isValidToken';
+
+export const refreshAuth = async (): Promise<void> => {
+  const accessToken = authService.getToken();
+  const refreshToken = authService.getRefreshToken();
+  if (!isValidToken(accessToken) && isValidToken(refreshToken)) {
+    await authService.renewToken();
+  }
```

Comment:
This part of the code might not get triggered properly ATM, I wrote some comments about it into the PR's description:

> ### Active renew based on user actions
> 
> Added some console logs to the active refresh, but couldn't get it to come up when testing. So this might not be working at the moment, but still, the silent renewal works. So actually we could perhaps even make the code simpler and remove the active renew completely... hmm.... 🤔

So this part definitely needs some looking into. The silent renew works according to my testing.

Reply:
Active renewal doesn't seem to work currently at all, moved that code to its own PR #271